### DISCUSSION TITLE
Battery Charge Limit set is reset by BIOS on boot.

### DISCUSSION THREAD

**Discussion by sreydray** [seerge/g-helper#3401] [General]
I have set my charge limit to 60%, but each time I boot up, the charging light goes red and it charges until G-helper reopens after boot. Also, if I am away for sometime keeping my PC at lock screen, it charges for a longer time as it seems Windows delays startup apps when on lock screen.

Is there a way to prevent battery from charging while it boots up?

**Comment by seerge** [comment]
@syedrey hello, this is how majority of devices work. They reset limit on reboot :) you can’t do much about that.

If you would run G-Helper as admin at least once - it would add a special startup task that starts before user login in windows and sets charge limit a bit earlier
